.github/dockerfiles/ov_build/ubuntu_22_04_x64_dpcpp/Dockerfile

```diff
@@ -0,0 +1,110 @@
+ENV APM=/opt/intel/oneapi/advisor/2024.2/perfmodels
+ENV CMAKE_PREFIX_PATH=/opt/intel/oneapi/tbb/2021.13/env/..:/opt/intel/oneapi/mkl/2024.2/lib/cmake:/opt/intel/oneapi/ipp/2021.12/lib/cmake/ipp:/opt/intel/oneapi/dpl/2022.6/lib/cmake/oneDPL:/opt/intel/oneapi/dnnl/2024.2/lib/cmake:/opt/intel/oneapi/dal/2024.5:/opt/intel/oneapi/compiler/2024.2
+ENV CMPLR_ROOT=/opt/intel/oneapi/compiler/2024.2
+ENV INFOPATH=/opt/intel/oneapi/debugger/2024.2/share/info
+ENV IPPROOT=/opt/intel/oneapi/ipp/2021.12
+ENV PYTHONPATH=/opt/intel/oneapi/advisor/2024.2/pythonapi
+ENV DALROOT=/opt/intel/oneapi/dal/2024.5
+ENV LIBRARY_PATH=/opt/intel/oneapi/tbb/2021.13/env/../lib/intel64/gcc4.8:/opt/intel/oneapi/mpi/2021.13/lib:/opt/intel/oneapi/mkl/2024.2/lib/:/opt/intel/oneapi/ippcp/2021.12/lib/:/opt/intel/oneapi/ipp/2021.12/lib:/opt/intel/oneapi/dpl/2022.6/lib:/opt/intel/oneapi/dnnl/2024.2/lib:/opt/intel/oneapi/dal/2024.5/lib:/opt/intel/oneapi/compiler/2024.2/lib:/opt/intel/oneapi/ccl/2021.13/lib/
+ENV IPPCRYPTOROOT=/opt/intel/oneapi/ippcp/2021.12
+ENV VTUNE_PROFILER_2024_DIR=/opt/intel/oneapi/vtune/2024.2
+ENV OCL_ICD_FILENAMES=/opt/intel/oneapi/compiler/2024.2/lib/libintelocl.so
+ENV CLASSPATH=/opt/intel/oneapi/mpi/2021.13/share/java/mpi.jar
+ENV LD_LIBRARY_PATH=/opt/intel/oneapi/tbb/2021.13/env/../lib/intel64/gcc4.8:/opt/intel/oneapi/mpi/2021.13/opt/mpi/libfabric/lib:/opt/intel/oneapi/mpi/2021.13/lib:/opt/intel/oneapi/mkl/2024.2/lib:/opt/intel/oneapi/ippcp/2021.12/lib/:/opt/intel/oneapi/ipp/2021.12/lib:/opt/intel/oneapi/dpl/2022.6/lib:/opt/intel/oneapi/dnnl/2024.2/lib:/opt/intel/oneapi/debugger/2024.2/opt/debugger/lib:/opt/intel/oneapi/dal/2024.5/lib:/opt/intel/oneapi/compiler/2024.2/opt/compiler/lib:/opt/intel/oneapi/compiler/2024.2/lib:/opt/intel/oneapi/ccl/2021.13/lib/
+ENV VTUNE_PROFILER_DIR=/opt/intel/oneapi/vtune/2024.2
+ENV MKLROOT=/opt/intel/oneapi/mkl/2024.2
+ENV NLSPATH=/opt/intel/oneapi/mkl/2024.2/share/locale/%l_%t/%N:/opt/intel/oneapi/compiler/2024.2/lib/compiler/locale/%l_%t/%N
+ENV PATH=/opt/intel/oneapi/vtune/2024.2/bin64:/opt/intel/oneapi/mpi/2021.13/bin:/opt/intel/oneapi/mkl/2024.2/bin/:/opt/intel/oneapi/dpcpp-ct/2024.2/bin:/opt/intel/oneapi/dev-utilities/2024.2/bin:/opt/intel/oneapi/debugger/2024.2/opt/debugger/bin:/opt/intel/oneapi/compiler/2024.2/bin:/opt/intel/oneapi/advisor/2024.2/bin64:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin
+ENV INTEL_PYTHONHOME=/opt/intel/oneapi/debugger/2024.2/opt/debugger
+ENV CPATH=/opt/intel/oneapi/tbb/2021.13/env/../include:/opt/intel/oneapi/mpi/2021.13/include:/opt/intel/oneapi/mkl/2024.2/include:/opt/intel/oneapi/ippcp/2021.12/include:/opt/intel/oneapi/ipp/2021.12/include:/opt/intel/oneapi/dpl/2022.6/include:/opt/intel/oneapi/dpcpp-ct/2024.2/include:/opt/intel/oneapi/dnnl/2024.2/include:/opt/intel/oneapi/dev-utilities/2024.2/include:/opt/intel/oneapi/dal/2024.5/include/dal:/opt/intel/oneapi/ccl/2021.13/include
```

Comment:
@akashchi do you know whether setupvars.sh can be used as ENTRYPOINT to setup oneAPI environment ? instead of manual copy-paste of all variables here